canonical/charmhub.io
# Add favicon

## Done
- Add website favicon
## How to QA
- Visit https://charmhub-io-1849.demos.haus/ , see favicon in browser tab. and hopefully soon on google search results.
## Testing
- [x] No testing required (explain why): Adds icon
